src/components/messageInputContainer.tsx

```
@@ -71,6 +82,77 @@ export const MessageInputContainer = ({ onChatProcessStart }: Props) => {
     }
   }
 
+  // Add initialization effect for cammicApp
+  useEffect(() => {
+    const initializeCammic = async () => {
+      console.log("Initializing cammicApp");
```

ESLint

[error] Replace `"Initializing·cammicApp");` with `'Initializing·cammicApp')`

(prettier/prettier)